Session-5/CurrencyExchangeApplication/ExtractCurrencyDB/ApiResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace CurrencyExchange
+{
+    public class ErrorType
```

Comment:
Why api response models are needed here? These should be part of CurrencyConverter library